### ISSUE TITLE
Schema validation errors should be part of the report (at least for html)

### ISSUE THREAD

**Issue by franky1964** [rangav/thunder-client-support#1246]
**Please describe feature/problem details and solution you'd like.**
Schema validation errors are only shown in console output but using TC in a CI/CD pipeline requires the details of the validation error  in html report al well.

Example:

```json
[
  {
    "instancePath": "",
    "schemaPath": "#/minItems",
    "keyword": "minItems",
    "params": {
      "limit": 10
    },
    "message": "must NOT have fewer than 10 items"
  }
]
```

**Describe alternatives you've considered**
As far as I know there exists no alternative

**Your Team Size Using TC:**
Expected team size is: 10-15

**Comment by rangav** [comment]
Thanks @franky1964 for the feedback, Will add it to the roadmap.

**Comment by rangav** [comment]
This feature is implemented and published to the marketplace, please update to `v2.13.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.13.0

Please let me know your feedback.
